conf/sample.ini

```diff
@@ -278,15 +278,18 @@
 # Login cookie name
 ;login_cookie_name = grafana_session
 
-# The maximum lifetime (duration) an authenticated user can be inactive before being required to login at next visit. Default is 7 days (7d). This setting should be expressed as a duration, e.g. 5m (minutes), 6h (hours), 10d (days), 2w (weeks), 1M (month). The lifetime resets at each successful token rotation 
-;login_maximum_inactive_lifetime_duration = 
+# The maximum lifetime (duration) an authenticated user can be inactive before being required to login at next visit. Default is 7 days (7d). This setting should be expressed as a duration, e.g. 5m (minutes), 6h (hours), 10d (days), 2w (weeks), 1M (month). The lifetime resets at each successful token rotation
+;login_maximum_inactive_lifetime_duration =
 
 # The maximum lifetime (duration) an authenticated user can be logged in since login time before being required to login. Default is 30 days (30d). This setting should be expressed as a duration, e.g. 5m (minutes), 6h (hours), 10d (days), 2w (weeks), 1M (month).
-;login_maximum_lifetime_duration = 
+;login_maximum_lifetime_duration =
 
 # How often should auth tokens be rotated for authenticated users when being active. The default is each 10 minutes.
 ;token_rotation_interval_minutes = 10
 
+# Set to true to disable the Grafana DB based login, useful if you are using LDAP, defaults to false
```

Comment:
Would need the same changes in defaults.ini and preferably also in the [docs](https://github.com/grafana/grafana/blob/master/docs/sources/administration/configuration.md#auth).

```suggestion
# Set to true to disable the Grafana built-in username/password authentication. Useful if you are using LDAP. Defaults to false.
```